tmt/steps/__init__.py

```diff
@@ -535,7 +531,7 @@ def show(self, keys: Optional[List[str]] = None) -> None:
             keys = self._common_keys + self._keys
         for key in base_keys + keys:
             # Skip showing the default name
-            if key == 'name' and self.name == tmt.utils.DEFAULT_NAME:
+            if key == 'name' and self.name.startswith(tmt.utils.DEFAULT_NAME):
                 continue
```

Comment:
Regarding to your irc comment:

>  if I use `name: default-client` in my fmf files, to complement my other guests, rhel8-client and rhel7-client, tmt will hide it from me even when it's been explicitly given by me

So what about using something like `re.match(tmt.utils.DEFAULT_NAME + r"-\d+$")` instead?

Reply:
Feels like too much effort expended in order to actually *hide* info from the user :)

Anyway, if you think it's worth the effort, I'd add something along these lines, to hide the "default family" of names.

Reply:
Np, how about this?

```patch
diff --git a/tmt/steps/__init__.py b/tmt/steps/__init__.py
index 879f742..f00ef48 100644
--- a/tmt/steps/__init__.py
+++ b/tmt/steps/__init__.py
@@ -531,7 +531,7 @@ class BasePlugin(Phase, metaclass=PluginIndex):
             keys = self._common_keys + self._keys
         for key in base_keys + keys:
             # Skip showing the default name
-            if key == 'name' and self.name.startswith(tmt.utils.DEFAULT_NAME):
+            if key == 'name' and tmt.utils.DEFAULT_NAME_FAMILY_PATTERN.match(self.name):
                 continue
             # Skip showing summary again
             if key == 'summary':
diff --git a/tmt/utils.py b/tmt/utils.py
index 41bfd24..95b0e9d 100644
--- a/tmt/utils.py
+++ b/tmt/utils.py
@@ -65,6 +65,7 @@ INDENT = 4
 
 # Default name and order for step plugins
 DEFAULT_NAME = 'default'
+DEFAULT_NAME_FAMILY_PATTERN = re.compile(rf'{DEFAULT_NAME}-\d+$')
 DEFAULT_PLUGIN_ORDER = 50
 DEFAULT_PLUGIN_ORDER_MULTIHOST = 10
 DEFAULT_PLUGIN_ORDER_REQUIRES = 70
```

Reply:
Sounds good to me!

Reply:
I'd add it as a follow up patch, I don't want to mess with the history of commits in this branch :/

Reply:
Ok, no problem.